submit-test-status/action.yml

```diff
@@ -33,58 +37,112 @@ runs:
       echo "-----"
       echo "Job name override: ${{ inputs.job_name_override }} (default job name: $GITHUB_JOB)"
       echo "BQ table name (for testing): ${{ inputs.big_query_table_name }}"
+      echo "Test results file path: '${{ inputs.test_results_file_path }}'"
+      echo "Test event record provided: ${{ inputs.test_event_record != '' }}"
+
+      # Show what the load input step will do
+      if [ -n "${{ inputs.test_results_file_path }}" ]; then
+        echo "Will use file path input method: ${{ inputs.test_results_file_path }}"
+      elif [ -n "${{ inputs.test_event_record }}" ]; then
+        echo "Will use inline data input method"
+      else
+        echo "INFO: No input detected - will skip CI Analytics submission (no test issues to report)"
+      fi
+      echo "-----"
+
+  - name: ensure that jq is available
+    uses: dcarbone/install-jq-action@v3.2.0
+
+  - name: load input
+    shell: bash
+    env:
+      TEMP_DIR: ${{ runner.temp }}/sts
+    run: |
+      mkdir -p $TEMP_DIR
```

Comment:
🔧 `shellcheck` would tell you to put quotes around this variable to prevent word splitting